java/test/jmri/jmrit/display/IconEditorWindowTest.java

```diff
@@ -106,9 +106,7 @@ public void testSensorEditor() throws Exception {
     @Test
     public void testRightTOEditor() throws Exception {
         Assume.assumeFalse(GraphicsEnvironment.isHeadless());
-        if (GraphicsEnvironment.isHeadless()) {
-            return; // can't Assume in TestCase
-        }
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
There are now two assumeFalse lines for isHeadless in this test.

Reply:
Thanks for catching that.